resources/script.js

```diff
@@ -13,12 +30,27 @@ function currentPosition() {
 function navigate(n) {
   var position = currentPosition();
   var numSlides = document.getElementsByClassName('slide').length;
-
-  /* Positions are 1-indexed, so we need to add and subtract 1 */
-  var nextPosition = (position - 1 + n) % numSlides + 1;
-
-  /* Normalize nextPosition in-case of a negative modulo result */
-  nextPosition = (nextPosition - 1 + numSlides) % numSlides + 1;
+  var nextPosition;
+  if (n === FIRST_SLIDE) {
+      nextPosition = 1;
+  } else if (n === LAST_SLIDE) {
+      nextPosition = numSlides;
+  } else {
+      if (! ALLOW_LOOPING) {
```

Comment:
I think cleaver just uses 2 spaces for tabs (which I don't really like these days), could we change these indents?